pkg/option/config.go

```diff
@@ -2500,6 +2509,7 @@ var (
 		AnnotateK8sNode:                 defaults.AnnotateK8sNode,
 		K8sServiceCacheSize:             defaults.K8sServiceCacheSize,
 		AutoCreateCiliumNodeResource:    defaults.AutoCreateCiliumNodeResource,
+		ExcludeNodeLabelPatterns:        []*regexp.Regexp{},
```

Comment:
Could this just be the default i.e nil slice?

Reply:
Makes sense. Removed this line now.